integrationtest/TargetProjects/NetStandardTestProject.XUnit/NetStandardTestProject.XUnit.csproj

```diff
@@ -12,7 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\TargetProject\TargetProject.csproj" Properties="TargetFramework=netstandard2.0" />
```

Comment:
If we remove the TargetFramework property, we are not testing netstandard. We should decide whether we actually need a separate netstandard integrationtest or not. And if so, we do need to set the TargetFramework property.